source/manual/sidekiq.html.md

```diff
@@ -0,0 +1,34 @@
+---
+owner_slack: "#govuk-2ndline"
```

Comment:
Should we merge the 'monitor' manual in here too?

Reply:
Good idea, I've pushed up a commit that does that.

Reply:
Great - thanks :-).